src/Cake.Common/Tools/NUnit/NUnitInternalTraceLevel.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Cake.Common.Tools.NUnit
+{
+    /// <summary>
+    /// Represents the level of detail at which NUnit should set internal tracing
+    /// </summary>
+    public enum NUnitInternalTraceLevel
```

Comment:
Can we assign values for these? `Off = 0` etc.

Reply:
Certainly. Change pushed.